package.json

```diff
@@ -11,9 +11,10 @@
     "build": "npx rimraf dist/ && tsc",
     "prettier": "prettier --write \"**/*\"",
     "lint": "tsc --noEmit && tsc -p ./test/tsconfig.json && tsc -p ./cypress/tsconfig.json",
-    "test": "npm run test:ut && npm run test:it",
-    "test:ut": "ts-mocha test/unitTest.ts",
-    "test:it": "ts-mocha test/integrationTest.ts --timeout 60000",
+    "test": "npm run test:ut && npm run test:e2e && npm run test:cmp",
+    "test:ut": "vitest run unit.test.ts",
+    "test:e2e": "vitest run e2e.test.ts --test-timeout 60000",
```

Comment:
the test commands have been updated. Please also update the [ci.yml](https://github.com/jimmux/cypress-fail-on-console-error/blob/main/.github/workflows/ci.yml#L23C35-L23C35) in order to run the pipeline.